var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -438,6 +445,17 @@ def define_tpl_enable(cmake_var, spec_var=None):
                 define('MPI_BASE_DIR', spec['mpi'].prefix),
             ])
 
+        # Cuda settings
+        options.append(define_tpl_enable('CUDA'))
+        if '+cuda' in spec:
+            cxx_flags.extend([
+                '--expt-extended-lambda'
```

Comment:
This is a NVCC flag: does it belong in the cxx_flags, or is that necessary due to the kokkos compiler wrapper?